corehq/apps/reports/standard/project_health.py

```diff
@@ -48,49 +49,81 @@ class MonthlyPerformanceSummary(jsonobject.JsonObject):
     active = jsonobject.IntegerProperty()
     performing = jsonobject.IntegerProperty()
 
-    def __init__(self, domain, month, performance_threshold, previous_summary=None):
+    def __init__(self, domain, month, users_filtered_by_group, performance_threshold, previous_summary=None):
         self._previous_summary = previous_summary
+        self._users_filtered_by_group = users_filtered_by_group
         self._next_summary = None
         self._base_queryset = MALTRow.objects.filter(
             domain_name=domain,
             month=month,
             user_type__in=['CommCareUser', 'CommCareUser-Deleted'],
         )
+        self._apply_group_filter = self._base_queryset.filter(
+            user_id__in=users_filtered_by_group,
+        )
         self._performing_queryset = self._base_queryset.filter(
             num_of_forms__gte=performance_threshold,
         )
+        self._performing_queryset_group_filtered = self._apply_group_filter.filter(
+            num_of_forms__gte=performance_threshold,
+        )
```

Comment:
can you clarify why you need both of these? can you not just apply the user filter if specified and otherwise not?